### DISCUSSION TITLE
Intro Thread :wave:

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#3] [General help]
We ❤️   our community members and want to know more about you. Tell us who are you, what you do, and one cool thing you’ve seen this week.

**Comment by phazonoverload** [comment]
I mean I work at Deepgram but I'll start! 

I'm a Developer Advocate which means I get to help developers discover, learn about, and build cool stuff with Deepgram. Outside of work I'm a shameless Disney adult and love boardgames (did you know there's an actual legit [Disney boardgame series](https://boardgamegeek.com/boardgame/256382/disney-villainous)?

**Reply by BekahHW** [reply]
Hey Kevin!

**Comment by BekahHW** [comment]
Hey! I'm Bekah, and I'm the Technical Community Builder at Deepgram! One cool thing I've seen, or rather experienced, is a creek not far from my house that I spent time walking in with one of my kids. It was beautiful and super relaxing!

**Reply by BekahHW** [reply]
Hi @khemit86! Support is working on the ticket you submitted. If you have other questions, take a look at our [Help Category](https://github.com/orgs/deepgram/discussions/13) and see if this is a good space for you to ask your question.

**Reply by BekahHW** [reply]
@khemit86, you might want to delete your API key and generate a new one since you exposed it in your post. If you have questions about keeping your API keys safe, check out this [blog post](https://blog.deepgram.com/protecting-api-key/).

**Comment by APMM-Deepgram** [comment]
Hi I'm Andrew, and I'm an account executive here at Deepgram. I help our customers test and build out and build cool things with Deepgram. Outside of work I like to get outside and take my dogs on a walk or a hike here in the beautiful PNW.

**Reply by BekahHW** [reply]
Hey Andrew! Great to see you in here 😉

**Comment by DamianG-DG** [comment]
Hi all, I'm Damian, I help customers find the best solution for their ASR needs.  Recently, I saw a cool demo of a company with an interactive avatar that not only answers your questions, but also reacts to your facial expressions!

**Comment by johnepop21** [comment]
Okay, I'm diving into the GitHub waters Discussion Thread.  This is a new experience for me. :-). 
I probably cannot help anyone code anything or further navigate GitHub, but I am very interested in using technologies to solve business problems in a way that recognizes the "caveman principle" *(footnote: a term you can learn about in many Michio Kaku books). 
My outside work interests include being outside for almost anything; horses, bikes, snow skiing, concert going and kid raising.  My big hobbies are reading about physics, astronomy, the JFK Assassination, and history.

**Comment by EliseJane** [comment]
Hey everybody, my name is El and I am a product engineer at Deepgram, building internal tools and customer proofs of concept to help push our collective vision forward. Besides work, I sing and I cook and I take care of my family and pets. Cool thing: I saw a video last week of how to make rustic homemade potato chips in the microwave that I am dying to try... looks too good to be true! I love potatoes.

**Reply by BekahHW** [reply]
Hey El! Drop that link to the potato chips here if you can 😋

**Reply by EliseJane** [reply]
found it! https://youtu.be/v514hwoC_fY

**Comment by shirgoldbird** [comment]
👋🏻 Shir here! I'm a product manager at DG working on our real-time transcription API. For fun, I play double bass, run festivals, and go on very long bike rides (100+ miles :) ).

Always happy to answer questions about how our product works. I'd love to hear your feedback about what we could be doing better, too!

**Comment by alexadlt** [comment]
Hi, I'm Alexa 👋🏼 I'm a Growth Marketer at Deepgram, primarily focused on sharing our amazing product with the world! When I'm not growth hacking a DG, you can find me eating my way around San Francisco, hanging out at the park, or on my next travel adventure 🗺.

**Reply by nmartin15** [reply]
Hi Nate, just someone who likes to explore néw AI technologies and see how I can use for my workflow .

**Comment by natalierutgers** [comment]
Hey hey 👋  I'm Natalie and yet another DGer helping to build our product and excitement around voice. This week I stumbled on this video of [Lizzo playing at the Library of Congress](https://twitter.com/librarycongress/status/1575208737052278817?s=20&t=C_BierZCgJ1BpG75PNKxyg). Loved it.

**Comment by kusum-ijari** [comment]
Heyy, I'm Kusum, a product designer at DG! Reach out with any feedback or questions you have on the design side of things!

This week I found out that this is [pixar's 404 page](https://www.pixar.com/dsf) and think that it's so adorable.

**Reply by BekahHW** [reply]
That 404 is so cute!!! I love it.

**Comment by hanoteau** [comment]
Hi here new deepgram user trying to adapt Kevin Lewis's S2Tpage code to help my deaf daughter understanding her professor at college/university. They speak French and despite activating the punctuation in the web socket calling deepgram I don't get any punctuation from the deepgram API while I get it if I transcribe from English. Is it a limitation for the French language model? If it is a limitation is there a time schedule to get the punctuation working or isn't it a short or mid term goal to achieve ? Please provide me with an honest answer.
Thanks in advance ty for the code

**Reply by BekahHW** [reply]
Hey @hanoteau! This is a question for one of our support engineers. You can contact them by emailing <redacted> or visiting the help center at [help.deepgram.com](http://help.deepgram.com/) to create a ticket. If you have any other questions that the DevRel team can answer, please post them in the [Q&A](https://github.com/orgs/deepgram/discussions/13) category. Wishing the best on your project!

**Comment by ielrekaa** [comment]
Hi there! I’m Ingrid. I’m a Product Designer and I always evaluate how things around me work. I think it’s a curse :sweat_smile:

This week I found out that Reddit can be used for user research. Maybe. Maybe not. I’m still learning. Also, this is far from new, but I can never get over this video: https://www.youtube.com/watch?v=IkOQw96cfyE

**Reply by zzega** [reply]
10/10 would work at a catvertizing agency

**Comment by jenny-deepgram** [comment]
Hi! I'm Jenny, Head of People for Deepgram. I'm also mom to two toddlers (age 2 & 3) and a novice home renovation DIY-er. When I'm at work, I spend a lot of time thinking about how to make Deepgram a truly awesome place for people to learn, connect and build. This weekend I am going with the family to see the Blue Angels - who continue to amaze me even though I've seen them fly dozens of times.

**Reply by michaeljolley** [reply]
The Blue Angels are amazing. One of their bases was a couple miles away from where we stayed at the gulf. Was cool to be in the ocean watching them practicing a few miles down the beach.

**Comment by zzega** [comment]
Hey everyone, I'm Sam, Deepgram's VP of Operations. I am nerdy about linguistics and love working on language-focused projects around Deepgram, whether it's how to curate data for a new non-English language model or how to label data for for one of our speech understanding models. There's so much cool research going on in this space, there's never a dull moment. When I'm not working, you might see me out and about San Francisco with a friendly husky named Cloud 😃

**Comment by cdyer3** [comment]
Hi there - I'm Chris, Deepgram's VP of Sales.  I love all of the cutting edge use cases that our customers execute on top of our speech rec.    I'm in the Bay Area and also coach high school basketball with my brother - we're on our 7th year now!

**Reply by zzega** [reply]
Coach Dyer ftw

**Reply by BekahHW** [reply]
I coached 6th-grade girls' bball this season! It might be my last season coaching bball though 🤪

**Comment by shadi-deepgram** [comment]
Hi Team - I'm Shadi Baqleh, COO for Deepgram. I love the amazing cool things we can help customers build. Especially in areas like automation and workflow.  Would love to continue learning more about AI, Clean Energy & Water, and more.

**Comment by tammydg** [comment]
Hi Everyone! I'm Tammy, Head of Talent at Deepgram and it's amazing to see this incredible community grow from 15 Deepgrammers to now over 100+ Deepgramers all over the world. This week I've seen so many cool things, one being the Warriors getting their '22 Championship ring! I'd love to connect if anyone is interested in chatting about all things recruiting related :)

**Comment by ghost** [comment]
Hello GitHub! My name is Evan and I'm a Product Manager here at Deepgram focusing on our on-premises solution. I was a Software Engineer for 6 years before I crossed over to the dark side and became a Product Manager, so I leverage my technical background to help me succeed in my role here at Deepgram. Additionally, I'm looking forward to the upcoming winter sports season so I can once again strap myself to my snowboard and throw myself down mountains.

**Comment by josencf** [comment]
Hello everyone!

My name is Jose, and I am an ML Developer Advocate at Deepgram. I've spent the last five years studying AI, particularly focusing on Natural Language Processing. I have a SWE background in the FinTech world, but the experience I'm proudest of is my work in helping teach classes at Stanford. I've taught an algorithms class, a data structures class, a Python class, and CS 101 (or 106A, as we called it on campus).

In my free time, I write stories, play piano, and dance badly 😸

**Comment by AlexNuggz** [comment]
Hi Everyone,

I'm Alex, I work in IT at a UK based law firm, just trialing the product and am amazed at the accuracy. We currently use BigHand for digital dictation which is blown out of the water by your product!

**Reply by SandraRodgers** [reply]
That's awesome Alex! I'm glad to hear that Deepgram is accurate for you; we really love to hear that! Please let me know if you need any help with anything or have questions about our API. 😀

Sandra

**Comment by djynnantonic** [comment]
Hello fellow Deepgram enthusiasts - I'm David Warhol, a decades-long video game developer (dating back to the Mattel Intellivision & NES days).  I'm always looking to integrate leading edge technologies and am fascinated with utilizing Deepgram's capabilities.  I'm currently creating a "Janet" 3D avatar (referencing the TV show 'The Good Place') that you can talk to about anything and it replies with spoken lip-synched ChatGPT answers (though getting those to be short may prove to be the biggest problem!).  Glad to brainstorm with anyone about anything related, from the old days of game development up to and including development services.

**Reply by SandraRodgers** [reply]
Hi David... Your "Janet" project sounds amazing! Would love to see a demo or blog post showing it off. One of our software engineers at Deepgram (@nikolawhallon) is also really into integrating Deepgram into games. He's written a couple blog posts about some of his projects:

https://blog.deepgram.com/deepgram-godot-tutorial/
https://blog.deepgram.com/deepgram-unity-tutorial/

Some of these current technologies like Deepgram and other AI tech have really opened up the possibilities of fun stuff to build, don't you think?

Sandra

**Reply by djynnantonic** [reply]
Thanks Sandra - I'll be sure to send you a link to the Janet when she's up.  My primary focus is currently on the tech, secondary adding production values, not sure if I'll want to go wide with the former.  And Niko has been helpful in the process.  Anyone who uses Godot is definitely into games!
